bindata/network/ovn-kubernetes/alert-rules-control-plane.yaml

```diff
@@ -66,3 +66,21 @@ spec:
       for: 10m
       labels:
         severity: warning
+    - alert: NorthBoundConnectionsError
+      annotations:
+        message: |
+          NorthBound DB has too many errors in connections.
+      expr: |
+          (ovn_db_cluster_inbound_connections_error_total{db_name="OVN_Northbound"} + ovn_db_cluster_outbound_connections_error_total{db_name="OVN_Northbound"}) > 100
```

Comment:
These values are counters, which means they are monotonic. So, you need to compute a rate of errors instead. Otherwise this alert will fire "forever", even if the rate of errors is one per hour, after 100 hours.

So the query might look something like

```
(rate(ovn_db_cluster_inbound_connections_error_total{db_name="OVN_Northbound"})[5m] + rate(ovn_db_cluster_outbound_connections_error_total{db_name="OVN_Northbound"})[5m] ) > 100
```

(This applies to the southbound alert as well)

Reply:
@squeed Good point..should have made it rate. Thanks.